### DISCUSSION TITLE
Is run_fpdb.py still used ?

### DISCUSSION THREAD

**Discussion by brunetton** [jejellyroll-fr/fpdb-3#32] [development]
Just a little doubt: on [dev-preview](https://github.com/jejellyroll-fr/fpdb-3/tree/dev-preview) branch, `run_fpdb.py` is still present. But I thought the "official" way to launch FPDB was following README and run `./fpdb.pyw`

Am I missing something ?

Thanks !

**Comment by jejellyroll-fr** [comment]
Hello Bruno,
i need to review this code again ... i am not sure to understand if it is usefull (perhaps not) ;)
